native/build.gradle

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Reply:
Fixed in [b617742](https://github.com/ballerina-platform/module-ballerinax-sap/pull/2/commits/b6177425121144b69a0b7c4a38381916f70d59fd)